pkg/skaffold/build/buildpacks/lifecycle.go

```diff
@@ -85,12 +83,6 @@ func (b *Builder) build(ctx context.Context, out io.Writer, a *latest.Artifact,
 }
 
 func runPackBuild(ctx context.Context, out io.Writer, localDocker docker.LocalDaemon, opts pack.BuildOptions) error {
-	// If out is not a terminal, let's make sure pack doesn't output with colors.
-	if _, isTerm := util.IsTerminal(out); !isTerm {
-		// pack uses heroku/color under the hood.
-		color.Disable(true)
```

Comment:
Oh, I guess a problem here is that we never re-enable color.

Reply:
Yes, the new code is more robust